op-service/txmgr/txmgr_test.go

```diff
@@ -389,14 +389,45 @@ func TestTxMgrNeverConfirmCancel(t *testing.T) {
 	}
 	h.backend.setTxSender(sendTx)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
waiting 10 seconds for a single unit test is excessive so I cut this in half. We'll still see multiple retries here since retry time is 1 second, if that's why this was waiting for so long.